cli/src/cmd/new.rs

```diff
@@ -84,10 +104,10 @@ fn initialize_for_lang(name: &str) -> Result<()> {
         }
     }
 
-    Ok(())
+    Ok(format!("Created contract {}", name))
```

Comment:
Why is returning a string needed here?

Reply:
So we can print a success message to the stdout: https://github.com/paritytech/ink/pull/177/files#diff-4c438f60247f494e2aee67fcce9e11e0R105